sources/packages/web/src/views/institution/InstitutionProfile.vue

```diff
@@ -12,42 +12,340 @@
+              <ValidatedInput property-name="primaryAddress.address1">
+                <Field
+                  name="primaryAddress.address1"
+                  label="Address 1"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-12">
+              <label for="primaryAddressAddress2">Address 2</label>
+              <ValidatedInput property-name="primaryAddress.address2">
+                <Field
+                  name="primaryAddress.address2"
+                  label="Address 2"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-6">
+              <label for="primaryAddressCity">City</label>
+              <ValidatedInput property-name="primaryAddress.city">
+                <Field
+                  name="primaryAddress.city"
+                  label="City"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-6">
+              <label for="primaryAddressPostalCode">Postal/ZIP code</label>
+              <ValidatedInput property-name="primaryAddress.postalCode">
+                <Field
+                  name="primaryAddress.postalCode"
+                  label="Postal/ZIP code"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-6">
+              <label for="primaryAddressProvinceState">Province/State</label>
+              <ValidatedInput property-name="primaryAddress.provinceState">
+                <Field
+                  name="primaryAddress.provinceState"
+                  label="Province/State"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-6">
+              <label for="primaryAddressCountry">Country</label>
+              <ValidatedInput property-name="primaryAddress.coutry">
+                <Field
+                  name="primaryAddress.coutry"
+                  label="Country"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+          </div>
+        </ContentGroup>
+        <HorizontalSeparator />
+        <h4>Privacy Disclaimer</h4>
+        <p>
+          Be continuing into the StudentAidBC Application you agree to
+          StudentAid BC’s Privacy policy.
+        </p>
+        <HorizontalSeparator />
+        <Button
+          type="submit"
+          label="Next"
+          icon="pi pi-save"
+          class="p-mt-4"
+          :disabled="isSubmitting"
+        ></Button>
```

Comment:
The look and feel in the screen shot underconversation page look odd. May be i am seeing something wrong, but will the look of text input boxes be similar to other screens in the applications?

Reply:
Textboxes and components, in general, are the same as the other pages in the portal.
Some text color may differ from the current wireframes, but it is not part of the requirements so far to them be exactly the same.